www/js/con-author.js

```diff
@@ -199,6 +223,73 @@ define([
 		 Handler for initial value input
 		 */
 		handleInitial: function(initial){
+
+			//Initial value handler for quantity node
+
+			//IniFlag contains the status and initial value
+			var modelType = this.getModelType();
+			console.log("model type is", modelType);
+			var tempIni = dom.byId(this.widgetMap.initial);
+			var tempInival = tempIni.value.trim();
+			console.log("temporary value is", tempInival);
+			var IniFlag = {status: undefined, value: undefined };
+			if(!((modelType === "given") && (tempInival == '') )){
```

Comment:
shouldn't we change the given models to authored model as it has been changed in model.js, to keep everything consistent.